fe/fe-core/src/main/java/org/apache/doris/mysql/MysqlColType.java

```diff
@@ -52,7 +52,8 @@ public enum MysqlColType {
     MYSQL_TYPE_BLOB(252, "BLOB"),
     MYSQL_TYPE_VARSTRING(253, "VAR STRING"),
     MYSQL_TYPE_STRING(254, "STRING"),
-    MYSQL_TYPE_GEOMETRY(255, "GEOMETRY");
+    MYSQL_TYPE_GEOMETRY(255, "GEOMETRY"),
+    MYSQL_TYPE_MAP(256, "MAP");
```

Comment:
we can use a larger value, eg. 400, instead of continous value to avoid conflict with new data type in mysql

Reply:
set 400 , done